cli/util/send.go

```diff
@@ -37,6 +40,17 @@ func sendTx(ctx *cli.Context) error {
 	if err != nil {
 		return cli.NewExitError(fmt.Errorf("failed to submit transaction to RPC node: %w", err), 1)
 	}
-	fmt.Fprintln(ctx.App.Writer, res.StringLE())
+	var aer *state.AppExecResult
+	if ctx.Bool("await") {
+		version, err := c.GetVersion()
+		if err != nil {
+			return cli.NewExitError(fmt.Errorf("failed to get version from RPC node: %w", err), 1)
+		}
+		aer, err = waiter.NewWaiter(c, version).Wait(tx.Hash(), tx.ValidUntilBlock, nil)
+		if err != nil {
+			return cli.NewExitError(fmt.Errorf("the transaction doesn't accepted to the block: %w", err), 1)
```

Comment:
It can be anything: RPC connection lost, websocket channel is closed, RPC server is broken. Thus, use formatted "failed to await transaction".